Resources/Prototypes/_Carpmosia/Recipes/Reactions/medicine.yml

```
@@ -29,6 +29,18 @@
     Fiber: 2
 
 - type: reaction
+  id: Barozine
+  impact: Medium
+  reactants:
+    Bruizine:
+      amount: 1
+    Copper:
+      amount: 1
+    Ethanol:
+      amount: 1
+  products:
+    Barozine: 1
+
   id: Tetracordrazine
```

```suggestion
- type: reaction
  id: Tetracordrazine
```
Got lost in the merge